README.md

```diff
@@ -45,6 +45,7 @@
 [how to install plasmapy]: https://docs.plasmapy.org/en/stable/install.html
 [installed Conda]: https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html
 [download and install Python]: https://wiki.python.org/moin/BeginnersGuide/Download
+[GitHub repository]: https://github.com/PlasmaPy/PlasmaPy
```

Comment:
Since the contents of the README file show up in places besides GitHub (like PyPI), it's good to have a link to our repo from the README as well (as suggested by a podcast).